### PR TITLE
Copyedit the "library" FAQ.

### PR DESCRIPTION
**Pull request by seh**
I made more progress through the documentation. If these small per-page patches are too annoying, let me know and I'll batch them.

### CODE REVIEW

File: content/faq/library.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
 date = "2015-05-12"
 +++
-The short answer is you can vendor source from one `gb` project to another -- just copy the code from your `$PROJECT/src/` into the downstream project.
+The short answer is yes: you can vendor source from one `gb` project to another&mdash;just copy the code from your `$PROJECT/src/` into the downstream project.
 
-Vendoring dependencies is similar, you'll need to merge the contents of your libraries `$PROJECT/vendor/src/` into the downstream's vendor directory.
+Vendoring dependencies is similar; you'll need to merge the contents of your libraries `$PROJECT/vendor/src/` into the downstream's vendor directory.
 
-Unfortunately `gb` cannot help you with this as Go projects don't have any concept of versions; there is no way you can look at the source of a Go package and ask it "what version are you" (assuming something like SemVer).
+Unfortunately `gb` cannot help you with this, as Go projects don't have any concept of versions; there is no way you can look at the source of a Go package and ask it "what version are you" (assuming something like [SemVer](http://semver.org/).
```
Comment (davecheney) [inline content/faq/library.md]: I regret even mentioning semver, please delete that parenthetical mention.

**Comment by davecheney** [comment]
LGTM with one minor nit, thanks.

**Comment by seh** [comment]
PTAL.

**Comment by davecheney** [comment]
Perfect, thanks.

**Comment by davecheney** [comment]
Not pushed to the site yet, I will do that soon

**Pull request merged**